sitespeed.io/scripts/machines.js

```diff
@@ -0,0 +1,10 @@
+const { constructURL } = require("../utils");
+
+const TIMEOUT = 120000;
+
+module.exports = async (context, commands) => {
+  await commands.measure.start("Machine list");
+  await commands.navigate(constructURL(context, "/machines"));
+  await commands.wait.byXpath("//*[text()='1000 Machines']", TIMEOUT);
+  return commands.measure.stop();
```

Comment:
If I understand correctly, we're waiting an arbitrary amount of time before stopping the measurement which does not guarantee we're measuring the entire page load cycle (until it's fully loaded and interactive). 2 minutes is probably sufficient, but could you think of any other ways of checking the page has fully loaded?

Reply:
Oooh, I missed that "small" detail. This should be perfectly fine then.